lib/rubocop/monkey_patches/config.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module RuboCop
+  class Config
+    DEFAULT_CHEF_VERSION = 15
+
+    # This is a copy of the #target_rails_version method from Rubocop
+    def target_chef_version
+      @target_chef_version ||=
+        if for_all_cops['TargetChefVersion']
+          for_all_cops['TargetChefVersion'].to_f
+        elsif target_chef_version_from_bundler_lock_file
+          target_chef_version_from_bundler_lock_file
+        else
+          DEFAULT_CHEF_VERSION
```

Comment:
I need to double check this. Since we define it in our config I think we can avoid the need for this constant and the constant updating of the constant

Reply:
I nuked this and just set it to 99